cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -503,10 +533,29 @@ Upgrading CDH
 
 .. _cloudera-release-specific-upgrade-notes:
 
-**Upgrading CDH (Cloudera Data Hub) 5.3 to 5.4**
-
-**Background:** CDH 5.3 ships with HBase 0.98 while CDH 5.4 ships with HBase 1.0. We support
-CDH 5.4 as of CDAP 3.1.0 - however, upgrading the underlying CDH version is only supported
+These steps cover upgrading the version of CDH of an existing CDAP installation.
+As the different versions of CDH can use different versions of HBase, upgrading from
+one version to the next can require that the HBase coprocessors be upgraded to the correct
+version. The table below lists the different coprocessors used with each version of CDH.
```

Comment:
suggestion: The table below lists the different coprocessor package names managed by CDAP for each version of CDH.